test/parallel/test-timers-socket-timeout-removes-other-socket-unref-timer.js

```diff
@@ -28,7 +28,7 @@ const server = net.createServer(function onClient(client) {
   }
 });
 
-server.listen(0, common.localhostIPv4, common.mustCall(() => {
+server.listen(0, 'localhost', common.mustCall(() => {
```

Comment:
```suggestion
server.listen(0, common.localhostIP, common.mustCall(() => {
```

Reply:
In that case it would only listen on either of the localhost addresses, but not dualstack. Don't we want dualstack?